standalone/test/unit_tests/util/unittest_tenth_of_degree.cpp

```diff
@@ -143,9 +151,9 @@ TEST(TenthOfDegreeTest, LargerThanOrEqualComparison)
 
 TEST(TenthOfDegreeTest, shouldThrowUnderflowErrorOnCastToIntOfValuesUnderZero)
 {
-  EXPECT_THROW(std::cout << static_cast<uint16_t>(util::TenthOfDegree(-1))
-                         << " - printed to not be unused. Should throw before!\n",
-               std::underflow_error);
+  uint16_t result UNUSED;  // Macro needed to suppress clang-tidy warning
+  EXPECT_THROW(result = static_cast<uint16_t>(util::TenthOfDegree(-1)), std::underflow_error)
```

Comment:
is the cast necessary? Why not use the original type?
```suggestion
  EXPECT_THROW(result = util::TenthOfDegree(-1), std::underflow_error)
```